src/app/orderManagement/orders/js/orders.config.js

```diff
@@ -18,10 +18,35 @@ function OrdersConfig($stateProvider) {
                     return ocParameters.Get($stateParams);
                 },
                 OrderList: function(ocOrdersService, OrderCloud, Parameters) {
-                    return ocOrdersService.List(Parameters);
+                    return ocOrdersService.List(Parameters)
+                        .then(function(orders) {
+                            if(orders.Items.length) {
+                                return orders;
+                            } else {
+                                return null;
+                            }
+                        });
```

Comment:
I'll check this again, but before it was just freezing up if there were no items being returned.